File: eRestorante/eRestorante.Models/Model/OrderDrinks.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eRestorante.Models.Model
{
    public partial class OrderDrinks
    {
        public int OrderDrinkId { get; set; }

        public int OrderQuantity { get; set; }

        public int? OrderId { get; set; }

        public int? DrinkId { get; set; }

    }
}

File: eRestorante/eRestoranteUI/erestorante_mobile/lib/models/orderDrinks.dart
import 'package:erestorante_mobile/models/role.dart';
import 'package:json_annotation/json_annotation.dart';

part 'orderDrinks.g.dart';

@JsonSerializable(explicitToJson: true)
class OrderDrinks{
  int? orderDrinkId;
  int? orderQuantity;
  int? orderId;
  int? drinkId;

  OrderDrinks(this.orderDrinkId, this.orderQuantity, this.orderId, this.drinkId);

  factory OrderDrinks.fromJson(Map<String, dynamic> json) => _$OrderDrinksFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$OrderDrinksToJson(this);
}